dist/electron-cookies.js

```diff
@@ -13069,20 +13069,15 @@ const { Cookie } = tough;
 // it's possible to send these cookies to a domain someone could register after the fact, but
 // for Heap, we need a parseable URI because internally we try to determine the right level to set
 // a cookie, rather than having a known set of public suffix domains.
-const FAKE_APP_URI = 'https://yourdomain.heap/';
+const FAKE_APP_URI = 'https://' + window.location.host;  
```

Comment:
Let's leave `yourdomain.heap` - it's fine since we're adding the new flag. I'm also not 100% sure you always get window.location.host on all hybrid web apps, so wary of using it.